@xen-orchestra/backups/_runners/_writers/IncrementalRemoteWriter.mjs

```diff
@@ -66,7 +66,7 @@ export class IncrementalRemoteWriter extends MixinRemoteWriter(AbstractIncrement
 
   async beforeBackup() {
     await super.beforeBackup()
-    return this._cleanVm({ merge: true })
+    return this._cleanVm({ merge: true, remove: true })
```

Comment:
IIRC, We did not do this before because it will delete unused VHDs which could be used as a base for a new delta.

Shouldn't we separate broken and unused VHDs deletion?

How do they interfere with chaining?

Reply:
Why would we chain a VHD with another from a deleted backup ? it will increase the number of entries and mess with the retention (that is how I detected this). 
We do a very light check in ncremental remote writer, , if the VHD is present and it can be open, we chain it. 

I would like to be more defensive here, since we still have an unclear edge case that lead to bad chaining in mirror backup